.travis.yml

```diff
@@ -68,11 +68,6 @@ env:
 
 matrix:
   include:
-    - python: 3.5
-      dist: trusty
-      # pytest-cov>=2.3.1 due to https://github.com/pytest-dev/pytest-cov/issues/124.
-      env:
-        - PINNEDVERS='-c requirements/testing/travis35.txt'
```

Comment:
Also remove this file (or rename it to be the 3.6 pins).

Reply:
We were pinning numpy to 1.10 on the other builds?

Reply:
It looks like the Travis 3.6 build was getting reqs from travis_all.txt, so I think this is OK...

Reply:
OK, see new file.  I'm not sure if the other pinned reqs in travis35.txt were required to get it to run on 3.5 or if they were min deps we were wanting to test.  I suspect the former, so I removed them.

Reply:
pytest and sphinx are pinned as minimum deps also.

Reply:
Why is that?  Are we testing that devs can use old Sphinx and pytest?  Or are those user facing pins?

Reply:
I suppose so; those are the versions we specify in the docs.

Reply:
Fixed, - wasn't sure if w need the pin on pytest-timeout so trying w/o